translate/api/TranslateV3Samples/TranslateV3BatchTranslateTextWithGlossaryAndModel.cs

```diff
@@ -0,0 +1,135 @@
+
+namespace TranslateV3Samples
+{
+    public static class TranslateV3BatchTranslateTextWithGlossaryAndModel
+    {
+        /// <summary>
+        /// Batch translate text with Model and Glossary
+        /// </summary>
+        /// <param name="targetLanguage">Required. Specify up to 10 language codes here.</param>
+        /// <param name="sourceLanguage">Required. Source language code.</param>
+        /// <param name="modelId">The models to use for translation. Map's key is target language
+        /// code.</param>
+        /// <param name="glossaryId">Required. Specifies the glossary used for this translation.</param>
+        public static void BatchTranslateTextWithGlossaryAndModelSample(string inputUri, string outputUri,
+            string projectId, string location, string targetLanguage,
+            string sourceLanguage, string modelId, string glossaryId)
+        {
+            TranslationServiceClient translationServiceClient = TranslationServiceClient.Create();
+
+            string modelPath = $"projects/{projectId}/locations/{location}/models/{modelId}";
+            string glossaryPath = $"projects/{projectId}/locations/{location}/glossaries/{glossaryId}";
+
+            BatchTranslateTextRequest request = new BatchTranslateTextRequest
+            {
+                ParentAsLocationName = new LocationName(projectId, location),
+                SourceLanguageCode = sourceLanguage,
+                TargetLanguageCodes =
+                {
+                    // Required. Specify up to 10 language codes here.
+                    targetLanguage,
+                },
+                InputConfigs =
+                {
+                    new InputConfig
+                    {
+                        GcsSource = new GcsSource
+                        {
+                            InputUri = inputUri,
+                        },
+                        MimeType = "text/plain",
+                    },
+                },
+                OutputConfig = new OutputConfig
+                {
+                    GcsDestination = new GcsDestination
+                    {
+                        OutputUriPrefix = outputUri,
+                    },
+                },
+                Models =
+                {
+                    // The models to use for translation. Map's key is target language code.
+                    { targetLanguage, modelPath },
+                },
+                Glossaries =
+                {
+                    { targetLanguage, new TranslateTextGlossaryConfig
+                        {
+                            Glossary = glossaryPath,
+                        }
+                    },
+                },
+            };
+            // Poll until the returned long-running operation is completed.
+            BatchTranslateResponse response = translationServiceClient.BatchTranslateText(request).PollUntilCompleted().Result;
+            // Display the translation for each input text provided
```

Comment:
Only one input text provided. This comment is misleading. And same in the rest of samples.